webapp/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultStudentGroupBatchService.java

```diff
@@ -1,5 +1,7 @@
 package org.opentestsystem.rdw.admin.service.impl;
 
+import com.amazonaws.services.s3.Headers;
```

Comment:
The LocalArchiveService and S3ArchiveService are both using that value to key off of... it seemed reasonable to me.  Objections?